docs/openshift_horizontal_scaling.md

```diff
@@ -34,11 +34,12 @@ doc](./async.md).
 ### Redis databases
 
 The standard 3scale deployment uses a single Redis pod and sets both the resque
-DB and the data DB in the same Redis process under different DB indexes. This is
-not supported by the async redis library, so we need to either place both DBs in
-the same Redis process or use two separate ones. To configure this, change the
-`CONFIG_REDIS_PROXY` and `REDIS_QUEUES_URL` envs in the listener and worker
-deployment configs.
+DB and the data DB in the same Redis process under different DB indexes. If an
```

Comment:
This change was mainly to fix the statement that DB indexes are not supported in async redis library. This feature was added here: https://github.com/3scale/apisonator/pull/390